src/views/Dashboard/Dashboard.tsx

```diff
@@ -25,25 +26,46 @@ const DEXWidget: React.FC = () => {
 };
 
 const Dashboard: React.FC = () => {
+  const isMobileLarge = useBreakpoint("<", "lg");
```

Comment:
a better naming scheme might be isScreenLg or isScreenSm, etc. nbd if you don't change it, the isMobile part got me thinking about it when it's large, since that's a desktop screen and not mobile.

also note here that isMobileLarge is going to be true when the screen is less than the large breakpoint.

if we need to extend the logic to match if the screen is currently a specific breakpoint size, then we can add the logic to usebreakpoint. the '.only' call seems to be what we need. see: https://mui.com/material-ui/customization/breakpoints/#theme-breakpoints-down-key-media-query

Reply:
additionally, isMobileSmall currently is just selecting if the breakpoint equals xs.

Reply:
i guess these don't work the way I thought they did which could explain why I was having trouble getting things layed out the way I wanted